1.9/usage/cli/command-reference/dcos-task/dcos-task-exec.md

```diff
@@ -0,0 +1,34 @@
+---
+post_title: dcos task exec
+menu_order: 0
+---
+
+# Description
+Launch a process inside of a task container.
+
+# Usage
+
+```bash
+dcos task exec [OPTION]
+```
```

Comment:
the current format of the CLI reference topics is to list a single `[OPTION]`, and then list the options in the table below (see https://dcos.io/docs/1.9/usage/cli/command-reference/dcos-node/dcos-node-ssh/). this was inspired by Docker docs: https://docs.docker.com/engine/reference/commandline/create/#options. i'm open to changing if you guys think it would be more useful to document this way (@tamar).